### ISSUE TITLE
encryption key is changing frequently

### ISSUE THREAD

**Issue by Honk2** [loqui/im#1058]
I am not sure if this an issue at all or just a consequence of some missing features/encryption in loqui.

I checked my contact status on iPhohe, WindowsPhone and Android. All note that Loqui is not supporting encryption and all and communication with we will not be encrypted (open lock smybol).
On iPhone the WA App frequently notes within the chat, that my encryption key (Sicherheitsnummer) has changed. This does not correlate with the installation of a new version, but happens sometimes daily and sometime less often.

Is this a hint, that something does not work or just that something is missing?

**Comment by cmeerw** [comment]
Can you correlate these key changes with anything happening on the Loqui side?

Does the "LOCAL REGISTRATION ID" change? Does Loqui send any type "retry" receipts? Or iq set "encrypt" requests to push new pre-keys to the server?

**Comment by Honk2** [comment]
By reading some logs, I can say that the encryption works with single contacts. Thus the message "not supporting encryption" seems to be due to missing group encryption.
I will now wait for the changing security number.

**Comment by Honk2** [comment]
Of cause, this issue did not happen again, since I opened this issue. Only when installing a new version which is expected.
also I just installed the latest source
- On iPhone and Android WA says the communication with me is now encrypted / all messages are e2e enxrypted!
- I can see the messages from windows phone users in group messages now, which faild with the previous source.
- A group with Androids only and me is now marked as totally encrypted, even without leaving and rejoining the group!

@cmeerw GREAT JOB! Thank you.
